apps/worker/src/transaction_notifier/notifier.service.helper.publishHandle.ts

```diff
@@ -13,7 +14,7 @@ export const handlePublishHandleTxResult = (txResultEvents: Vec<EventRecord>): R
   let debugMsg;
 
   txResultEvents.forEach((record) => {
-    const { method, data, section } = record.event;
+    const { method, data, section }: { method: string; data: IEventData; section: string } = record.event;
```

Comment:
nit: this type def is used multiple times, probably good to make a `type` declaration for it